packages/core/src/ui/Base/ThemeProvider.tsx

```diff
@@ -0,0 +1,2 @@
+import { ThemeProvider } from "next-themes";
+export { ThemeProvider };
```

Comment:
I don't think a separate file is required here, we can directly export ThemeProvider in index.